docs/community.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Community
+type: docs
+menu: thanos
```

Comment:
maybe `contributing` instead? main menu is bloated already

Reply:
I've moved under contributing :)